apps/challenger/package.json

```diff
@@ -10,20 +10,20 @@
   "author": "daveroga",
   "license": "CC0-1.0",
   "dependencies": {
+    "@polygon-nightfall/common-files": "^0.0.1",
     "async-mutex": "^0.3.2",
     "body-parser": "^1.19.2",
-    "common-files": "file:../common-files",
+    "cls-hooked": "^4.2.2",
```

Comment:
Hey @pawelgrzybek, I have a question: the deps `express-fileupload`, `cls--hooked`, `uuid`, `pino`and `pino-pretty` are needed everywhere because the old way we used the `common-files` package. Now that we are moving to this new way, isn't it the time for removing all that possibility unnecessary deps?

Reply:
Let's have a quick call about it, because I am not sure If I understand. But trust me friend, if I can remove something, I will. I love removing code!

Reply:
I see your point now. You are absolutely right, we don't need these here anymore. I would prefer to fix this as part of the other issues and other ticket. I would prefer to keep this PR scoped to replacing common-files, and I will open another one to identify unused packages. Does it sound like a good plan to you my friend?

Reply:
> I see your point now. You are absolutely right, we don't need these here anymore. I would prefer to fix this as part of the other issues and other ticket. I would prefer to keep this PR scoped to replacing common-files, and I will open another one to identify unused packages. Does it sound like a good plan to you my friend?

Amazing plan! 😊

Reply:
I also love removing code. I'd remove it all, but then stuff wouldn't work